r2-netty/src/main/java/com/linkedin/r2/transport/http/client/HttpClientFactory.java

```diff
@@ -1121,7 +1120,7 @@ private TransportClient getClient(Map<String, ? extends Object> properties,
     filters = filters.addLastRest(disruptFilter);
     filters = filters.addLast(disruptFilter);
 
-    client = new FilterChainClient(client, filters);
+    client = new FilterChainClient(client, filters, _filters);
```

Comment:
As mentioned in the FilterChainClient file, instead of passing down the common _filters down, it would be more clear if we introduce some kind of flag in the existing filter to identify global vs local filters

Reply:
Sure, we can.